client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -183,6 +184,20 @@ UndocumentedSite.prototype.setOption = function( query, callback ) {
 	);
 }
 
+UndocumentedSite.prototype.postCounts = function( options, callback ) {
+	var type,
+		query = merge( {
```

Comment:
`Object.assign` would work just as well here without the additional dependency, since this is a shallow object.

(Though an argument could be made that [Lodash's implementation is faster](https://jsperf.com/assign-lodash-vs-core-js), in which case `lodash/assign` is still more appropriate)